sklearn/utils/multiclass.py

```diff
@@ -53,11 +53,11 @@ def unique_labels(*ys):
 
     Parameters
     ----------
-    *ys : array-likes
+    *ys : array-like
```

Comment:
Since `ys` can be multiple arary-likes, I think using plural is okay here.

Reply:
Thanks, fixed in new commit.